test/fixtures/usage-sigular-help.fixture

```diff
@@ -0,0 +1,35 @@
+A test command to illustrate help and usage information. This
+command has a very long description that spans multiple lines.
+It should be broken up accordingly, depending on the longest line of text.
+==========================================================================
+usage -a 'fake' --verbose
+
+Options:
+
+  -i, --interactive, --no-interactive   <boolean> [false]             Use the interactive propmts
+  --color, --no-color                   <boolean> [true]              Enable ANSI color in output
+  --my_required                         <string>                      This flag is required
+  --name                                <string>                      This is my name
+  --nickname                            <string>                      This is my nickname
+  --city                                <string>                      This is my city
+  --state                               <string>                      This is my state
+  --zip                                 <number>                      This is my zip code
+  --country_code                        <number>                      This is my country code
+
+
+  <...>: input type
+  *: repeatable flags
+  [...]: default values
+
+
+Required Flags:
+  * my_required
+
+Mutually Inclusive Flags:
+  * city, state
+
+Mutually Exclusive Flags:
+  * zip, country_code
```

Comment:
You are the only person on the planet that uses underscores in command line flags 😏

Reply:
heh, just a habit when doing props on objects.  I'll totally fix it.